sema/commons/j2/j2_functions.py

```
@@ -156,6 +157,111 @@
+def _auto_str_to_formatted_number(content: str, quote: str) -> str | None:
+    testcontent = content.strip().lower()
+    if testcontent[0] in ["-", "+"]:
+        testcontent = testcontent[1:]
+    if testcontent.isdigit():
+        return xsd_format_integer(content, quote)
+    if testcontent.replace(".", "", 1).isdigit():
+        return xsd_format_double(content, quote)
+    return None
+
+
+def xsd_auto_format_date(content: Any, quote: str, *_: Any) -> str:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `content`

(ANN401)